entrypoints/example.ipynb

```diff
@@ -22,6 +25,15 @@
     "print(cred)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "token = spark_utils.credentials.getToken(\"Synapse\")"
```

Comment:
You wanna add an `assert token` just to check it's non-empty (and therefore functional)?